src/excmds.ts

```diff
@@ -3281,32 +3281,33 @@ export async function yankimage(url: string): Promise<void> {
 
 /** Change active tab.
 
-    @param index
-        Starts at 1. 0 refers to last tab of the current window, -1 to penultimate tab, etc.
+    @param id
+        A bare number means the current window is used.
 
-        "#" means the tab that was last accessed in this window
+        A string following the following format: "[0-9]+.[0-9]+" means the first number being the index of the window that should be selected and the second one being the index of the tab within that window. [[taball]] has completions for this format.
 
-    This is different from [[taball]] because `index` is the position of the tab in the current window.
+        "#" means the tab that was last accessed in this window
  */
 //#background
-export async function tab(index: number | "#") {
-    return tabIndexSetActive(index)
-}
-
-/** Change active tab.
-
-    @param id
-        A string following the following format: "[0-9]+.[0-9]+", the first number being the index of the window that should be selected and the second one being the index of the tab within that window.
+export async function tab(id: string) {
```

Comment:
This should probably be `...id: string[]` so e.g. `:tab BBC news` works and doesn't silently drop the news